### ISSUE TITLE
Stuck smoking rack

### ISSUE THREAD

**Issue by PatrikLundell** [CleverRaven/Cataclysm-DDA#57439]
### Describe the bug

In the Experimental version downloaded yesterday smoking racks no longer finish their task.

### Steps to reproduce

1. Find a dead pig
2. Bring it back to base and butcher it.
3. Start to smoke the meat in the smoking racks.
4. Find that one rack is stuck on smoking tomatoes from sometimes earlier (either morning or previous day).
5. Shrug and toss as much as you can into the other two racks.
6. Render fat, take care of hides, and then go to sleep.
7. Check the racks the next morning, and find they're still not done.

### Expected behavior

Smoking takes the same time as it used to (8 hours?).

### Screenshots

The tomatoes still being processed from the previous morning or the day before that.
![image](https://user-images.githubusercontent.com/22739822/166917142-d160ecd7-02fe-4536-bd07-f7be7e37fb0e.png)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19043.1645 (21H1)
- Game Version: f97c81e [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions]
]

### Additional context
Edit: I can also mention that no smoke is generated from the smoking racks. I believe smoke was generated when I lit them up.

Edit 2: 
I've updated to Game Version: 30c8dc3 [64-bit], and the smoking racks are still stuck. However, a smoking rack I lit up on another site using the original report version had finished smoking when I checked in on it. I guess the stuff smoking in my base camp will have to be written off.

Edit 3: I've killed off the processing twice now, with the latest attempt after having slept through the night and not saved and reloaded (which I had in the previous attempt). I have a nasty suspicion that it might be the work to accept multiple types of coal that's caused it, as the broken smoking racks are camp ones (and thus being registered as pseudo tools), while the off site one was built manually. I'm going to see if I can salvage the food, haul it to the other site, and smoke it there.

**Comment by RoyBerube** [comment]
Was the basecamp smoker ever loaded with coal? That might be the problem - the interaction menu only recognizes charcoal. Any coal inside a furniture smoker cannot be unloaded and will not burn.

**Comment by PatrikLundell** [comment]
No. Only ever charcoal. I haven't had any companion crafting done that might have used the racks either.

**Comment by SamGondra** [comment]
You are probably best of if you upload the safe (best standing beside the broken smokers).

**Comment by PatrikLundell** [comment]
That is an option only for a very short time after a game's start. After that the save gets too large to be accepted for upload even with maximum compression.

**Comment by andrei8l** [comment]
Use a third party website or just delete all the other submaps and map memory from your save. Map memory is saved in the `xxxx.mm1` folder and you can find the submap coordinates with the long-range teleport debug option.

**Comment by PatrikLundell** [comment]
Link to save with the issue: https://www.dropbox.com/s/b0qx5yz9dn2m29w/Weippe.zip?dl=0 where the map cache has been removed. I still can't upload the 70 MB file here ("is not included in the list").

The two southern smoking racks were lit at 12:40, so the 6 hours mentioned when lighting them should have been reached at 18:40. In the main game I've progressed until the next day, and the racks are still stuck there.
